src/crimson/net/ProtocolV1.cc

```diff
@@ -595,6 +595,45 @@ void ProtocolV1::start_accept(SocketFRef&& sock,
 
 // open state
 
+seastar::future<> ProtocolV1::write_messages(std::queue<MessageRef>& msgs)
+{
+  bufferlist bl;
+  size_t len = msgs.size();
+  while (!msgs.empty()) {
+    auto msg = msgs.front();
```

Comment:
Maybe there is no need to bump up the counter of `MessageRef`. We could `::pop()` after processing current message.

Reply:
done.